test/distance/edge-filters-test.cpp

```diff
@@ -0,0 +1,115 @@
+#include <gtest/gtest.h>
+
+#include "test/common/common.hpp"
+#include "src/distance/edge.hpp"
```

Comment:
In test cases, as a convention, please try to follow the conventional grouping of import statements, seperated by spaces:
1. External C Libraries
2. External C++ Libraries
3. Our Libraries
4. Our Test Libraries

You'll also find this convention mentioned [here](https://www.notion.so/huskysat/C-Software-Style-24b4a4f5c8e081abaa17da8bf58cc52b?source=copy_link#24b4a4f5c8e0819796b5ee2f2b876c51)